tests/kas_messaging.spec.ts

```diff
@@ -0,0 +1,107 @@
+import { test, expect } from '@playwright/test';
+import login from '@lib/auth';
+import { config } from '@lib/config';
+import {
+  navigateToKafkaList,
+  deleteKafkaInstance,
+  createKafkaInstance,
+  getBootstrapUrl,
+  navigateToAccess,
+  navigateToConsumerGroups,
+  grantProducerAccess,
+  grantConsumerAccess
+} from '@lib/kafka';
+import { navigateToKafkaTopicsList, createKafkaTopic, navigeToMessages } from '@lib/topic';
+import { KafkaConsumer, KafkaProducer } from '../lib/clients';
+import { strict as assert } from 'assert';
+import { createServiceAccount, deleteServiceAccount, navigateToSAList } from '@lib/sa';
+
+const testInstanceName = config.instanceName;
+const testTopicName = `test-jstejska`;
+const testServiceAccountName = 'jstejska-test';
```

Comment:
Don't forget to change these strings before you get the PR to "ready for review" state, please. Nice easter egg though 😄